packages/api/tests/lib/test-granule-delete.js

```diff
@@ -497,89 +404,3 @@ test.serial('deleteGranuleAndFiles() will not delete granule or S3 files if the
     s3Buckets.public.name,
   ]));
 });
-
-test.serial('deleteGranuleAndFiles() does not require a PostgreSQL granule', async (t) => {
```

Comment:
A modified version of this test may still be valid. It looks like we can call `deleteGranuleAndFiles` without a pgGranule if we pass an ES granule.

Reply:
Good catch 👍🏻

Reply:
Take a look at 727b8b1267

Reply:
Let's talk about this in an architecture meeting.